i18n/fr.json

```
@@ -17,16 +17,16 @@
   "disableAutoRefreshDesc": "Désactive l'affichage en temps réel des tweets dans votre fil d'actualités. <br/>Ne vous attendez pas à grand-chose, cela ne fonctionne pas tout le temps.",
   "forceLatest": "Forcer les tout derniers Tweets",
   "forceLatestDesc": "Activer le mode « tout derniers Tweets » n'est plus permanent. En activant cette option, le site revient automatiquement à ce mode.",
-  "hideMessageBox": "*NEW* Hide Message Box",
-  "hideMessageBoxDesc": "*NEW* Hides the new, all time present message box in the bottom right corner.",
-  "hideTranslateTweetButton": "*NEW* Hide Translate Tweet Button",
-  "hideTrends": "*NEW* Hide Trends",
-  "hideWhoToFollow": "*NEW* Hide Who To Follow",
+  "hideMessageBox": "Cacher la boîte de Messages,
```

leaving this out would break everything :S
```suggestion
  "hideMessageBox": "Cacher la boîte de Messages",
```